services/QuillLessonsServer/bootstrap.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+echo "Installing script dependencies..."
+brew install jq
+
+echo "Installing RethinkDB for local use..."
+brew update
+brew install rethinkdb
+
+echo "Setting up local RethinkDB in prep to load seed data..."
+./rethink_local.sh start
+echo -n "Please provide the path to your RethinkDB seed data dump: "
+read rethink_dump_path
+echo "Loading seed data into local RethinkDB instance..."
+$(rethinkdb restore --force $rethink_dump_path)
```

Comment:
What does `--force` do if you already have data in the DB? Will you end up with duplicate data?

Reply:
Testing locally suggests that it does not increase the size of the DB to run multiple times.  Looks like all the primary keys just get updated, maybe.